app/src/main/java/com/globant/weatherapp/data/entity/FiveDaysWeather.kt

```diff
@@ -1,4 +1,4 @@
-package com.globant.weatherapp.data.entities
+package com.globant.weatherapp.data.entity
 
 data class FiveDaysWeather (
```

Comment:
Why you put this name? Try to use another namely because If in a future you decide to change and show six day extended that class would be change...

Reply:
Yes, refactored!